packaging/windows/WiXTools/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilationOptions": {
+      "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "WiX": "3.10.0.2103-pre1"
```

Comment:
Ignore the first part of my earlier comment
>  It does remove paths (added after installation) from %path% after uninstallation. 

And yes I confirm that uninstall does *not* affect %path% other than removing dotnet path.